src/spdx2/agent/template/spdx2tv-file.twig

```diff
@@ -21,6 +21,14 @@ LicenseConcluded: {{ concludedLicense }}
 {% else %}
 LicenseConcluded: NOASSERTION
 {% endif %}
+{% if licenseCommentState %}
+{% if licenseComment is empty %}
+LicenseComment: NOCOMMENT
```

Comment:
When no license comment is defined, I'm not sure that NOCOMMENT should be used.
Shouldn't we simply skip the line ?

While the NOASSERTION is well defined in the SPDX specs, nothing is said on NOCOMMENT.
https://spdx.org/spdx-specification-21-web-version#h.28h4qwu

Reply:
OK for me to skip the line.